common/helpers.js

```diff
@@ -1,6 +1,6 @@
 const url = require('url');
 const config = require('../common/config');
-
+const sleep = require('util').promisify(setTimeout);
```

Comment:
I believe this will be fine. https://nodejs.org/api/util.html#util_util_promisify_original mentions the pitfall with `this`, but that isn't a problem here.